File: backend/app/Models/Item/Item.php
<?php

namespace App\Models\Item;

use Eloquent;
use Illuminate\Database\Eloquent\Builder;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsTo;

/**
 * @mixin Eloquent
 * @mixin Builder
 */
class Item extends Model
{
  protected $fillable = [
    'title',
    'is_activated',
    'preview_url',
    'description',
    'price',
    'item_category_id',
  ];
  protected $casts = [
    'price' => 'float'
  ];
  public function itemCategory(): BelongsTo
  {
    return $this->belongsTo(ItemCategory::class);
  }
}

File: UserPanel/Models/Item.cs
﻿using System.Text.Json.Serialization;
namespace UserPanel.Models;
public class Item
{
    [JsonPropertyName("id")] public required ulong Id { get; set; }
    [JsonPropertyName("title")] public required string Title { get; set; }
    [JsonPropertyName("is_activated")] public required bool IsActivated { get; set; }
    [JsonPropertyName("preview_url")] public required string PreviewUrl { get; set; }
    [JsonPropertyName("description")] public string? Description { get; set; }
    [JsonPropertyName("price")] public required decimal Price { get; set; }
    [JsonPropertyName("item_category")] public ItemCategory? ItemCategory { get; set; }

}